client/src/infographic/BubbleMenu.tsx

```diff
@@ -12,21 +12,33 @@ import { is_a11y_mode } from "src/core/injected_build_constants";
 
 import "./BubbleMenu.scss";
 
-const BubbleMenu = ({ items, active_item_id }) => {
+interface ItemProps {
```

Comment:
Did the bubble menu stuff get mixed in from a different branch? If not, I'd recommend splitting it out from this branch, I feel like `TreeMap` TSing could drag on for a bit.

Reply:
I won't review this file for now.